client/network_inspect.go

```diff
@@ -1,26 +1,23 @@
 package client
 
 import (
-	"bytes"
 	"context"
-	"encoding/json"
-	"io"
 	"net/url"
 
 	"github.com/moby/moby/api/types/network"
 )
 
-// NetworkInspect returns the information for a specific network configured in the docker host.
-func (cli *Client) NetworkInspect(ctx context.Context, networkID string, options NetworkInspectOptions) (network.Inspect, error) {
-	networkResource, _, err := cli.NetworkInspectWithRaw(ctx, networkID, options)
-	return networkResource, err
+// NetworkInspectResult contains the result of a network inspection.
+type NetworkInspectResult struct {
+	Network network.Inspect
```

Comment:
This one is slightly awkward but I don't have a good solution for that; the `network.Inspect` type is already a composite that embeds `Network`; you can access `response.Network.ID` directly, but of course also through `response.Network.Network.ID`